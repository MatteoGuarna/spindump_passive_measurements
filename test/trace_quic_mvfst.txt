simple test with mvfst version 0xfaceb002
