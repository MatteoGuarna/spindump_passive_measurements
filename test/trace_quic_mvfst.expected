QUIC 127.0.0.1 <-> 127.0.0.1 08ed7508907993a1-ec7508907993a1e1 (57257:9999) at 1585323603097074 new starting packets 0 0 bytes 0 0 note "No rsp,v.fbmv,no spin"
QUIC 127.0.0.1 <-> 127.0.0.1 08ed7508907993a1-08400000570df27a (57257:9999) at 1585323603097604 change starting packets 1 0 bytes 1280 0 bandwidth 1280 0 note "No rsp,v.fbmv,no spin"
QUIC 127.0.0.1 <-> 127.0.0.1 08ed7508907993a1-08400000570df27a (57257:9999) at 1585323603097604 measurement up right 530 packets 1 0 bytes 1280 0 bandwidth 1280 0 note "No rsp,v.fbmv,no spin"
QUIC 127.0.0.1 <-> 127.0.0.1 ed7508907993a1e1-08400000570df27a (57257:9999) at 1585323603097711 packet up responder length 165 packets 1 2 bytes 1280 240 bandwidth 1280 240 note "V.fbmv,no spin"
QUIC 127.0.0.1 <-> 127.0.0.1 ed7508907993a1e1-08400000570df27a (57257:9999) at 1585323603097730 packet up responder length 779 packets 1 3 bytes 1280 1019 bandwidth 1280 1019 note "V.fbmv,no spin"
QUIC 127.0.0.1 <-> 127.0.0.1 ed7508907993a1e1-08400000570df27a (57257:9999) at 1585323603097880 packet up initiator length 75 packets 2 3 bytes 1355 1019 bandwidth 1355 1019 note "V.fbmv,no spin"
QUIC 127.0.0.1 <-> 127.0.0.1 ed7508907993a1e1-08400000570df27a (57257:9999) at 1585323603098337 packet up initiator length 113 packets 3 3 bytes 1468 1019 bandwidth 1468 1019 note "V.fbmv,no spin"
QUIC 127.0.0.1 <-> 127.0.0.1 ed7508907993a1e1-08400000570df27a (57257:9999) at 1585323603098479 packet up responder length 74 packets 4 4 bytes 1535 1093 bandwidth 1535 1093 note "V.fbmv,no spin"
